### ISSUE TITLE
NVDA fails to display the connect dialog.

### ISSUE THREAD

**Issue by stickbear2015** [NVDARemote/NVDARemote#206]
The following error appears in the logs when i attempt to activate the connect dialog.
Note: I've attempted to uninstall and reinstall nvda, and to revert to factory defaults.
ERROR - unhandled exception (01:07:04.282):
Traceback (most recent call last):
  File "C:\Users\stick\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\__init__.py", line 285, in do_connect
  File "C:\Users\stick\AppData\Roaming\nvda\addons\remote\globalPlugins\remoteClient\configuration.py", line 31, in get_config
  File "configobj\__init__.pyo", line 1199, in __init__
  File "configobj\__init__.pyo", line 1292, in _load
ParseError: Invalid line ('\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00') (matched as neither section nor keyword) at line 1.
WARNING - eventHandler._EventExecuter.next (01:07:04.332):
Could not execute function event_becomeNavigatorObject defined in globalPlugins.Unspoken module; kwargs: {'isFocus': True}
Traceback (most recent call last):
  File "eventHandler.pyo", line 100, in next
TypeError: event_becomeNavigatorObject() got an unexpected keyword argument 'isFocus'
INFO - globalCommands.GlobalCommands.script_navigatorObject_

**Comment by TylerK93** [comment]
I'm not sure how that happened, but look at your NVDA Remote config file. It looks like Configobj is failing to parse some aspect of the configuration file because it's seeing an unexpected null string sequence on the first line of whatever file it is attempting to read. What NVDA version, and what version of the add-on?

**Comment by stickbear2015** [comment]
I’m running NVDA 2019.1.1 and Remote version 2.2.
This even happens on a clean install of the remote addon.

**Comment by tspivey** [comment]
Delete %appdata%\nvda\remote.ini.

**Comment by stickbear2015** [comment]
this has solved the issue. I'm marking it as closed.
